modules/effects/arrow-effects/src/main/kotlin/arrow/effects/IOParallel.kt

```diff
@@ -18,7 +18,7 @@ fun <A, B, C> IO.Companion.parallelMapN(ctx: CoroutineContext, ioA: IO<A>, ioB:
     cbf.unsafeGet().forEach { it(result) }
   }
 
-  return IO.async { cb: (Either<Throwable, C>) -> Unit ->
+  return IO.async { _, cb: (Either<Throwable, C>) -> Unit ->
```

Comment:
Cancellation for Parallels needs to be setup here.

Reply:
`IOConnection` is not exposed through Async anymore, mimicking cats. Further explanations in a comment in the end.